src/data/utils/dataUtils.js

```diff
@@ -24,7 +24,9 @@ export const getTpaProvider = (tpaHintProvider, primaryProviders, secondaryProvi
     secondaryProviders.forEach((provider) => {
       if (provider.id === tpaHintProvider) {
         tpaProvider = provider;
-        isSecondaryProvider = true;
+        if (provider.skipHintedLogin) {
+          isSecondaryProvider = true;
```

Comment:
Since this is not necessarily a secondary provider we should change the variable name to `skipHintedLogin` or `redirectToIdp` etc.